### PR TITLE
test capi: Stroke Radial Gradient

### PR DESCRIPTION
**Pull request by mmaciola**

### CODE REVIEW

**Review by mihashco — APPROVED** [review]

File: test/capi/capiRadialGradient.cpp [hunk 1]
```diff
@@ -133,4 +136,46 @@ TEST_CASE("Set/Get gradient spread", "[capiRadialGradient]")
...
     REQUIRE(tvg_gradient_del(gradient) == TVG_RESULT_SUCCESS);
     REQUIRE(tvg_gradient_del(NULL) == TVG_RESULT_INVALID_ARGUMENT);
-}
\ No newline at end of file
+}
+
+TEST_CASE("Stroke Radial Gradient", "[capiRadialGradient]")
+{
+    Tvg_Paint *shape = tvg_shape_new();
+    REQUIRE(shape != NULL);
```
[the remark this answers is not in the document]
Comment (mmaciola) [reply test/capi/capiRadialGradient.cpp]: done

File: test/capi/capiRadialGradient.cpp [hunk 2]
```diff
@@ -133,4 +136,46 @@ TEST_CASE("Set/Get gradient spread", "[capiRadialGradient]")
...
+
+    REQUIRE(tvg_gradient_get_color_stops(gradient_ret, &color_stops_ret,  &color_stops_count_ret) == TVG_RESULT_SUCCESS);
+    REQUIRE(color_stops_ret);
+    REQUIRE(color_stops_count_ret == 2);
+
+    float cx, cy, radius;
+    REQUIRE(tvg_radial_gradient_get(gradient_ret, &cx, &cy, &radius) == TVG_RESULT_SUCCESS);
+    REQUIRE(cx == 10.0);
+    REQUIRE(cy == 15.0);
+    REQUIRE(radius == 30.0);
```
Comment (JSUYA) [inline test/capi/capiRadialGradient.cpp:178]: You can use Approx class for float point comparison.
(https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#floating-point-comparisons)

**Review by JSUYA — COMMENTED** [review]
Please check my comment

File: test/capi/capiRadialGradient.cpp (hunk 2 above)
Comment (mmaciola) [reply test/capi/capiRadialGradient.cpp]: good catch. Done

**Pull request merged**